generators/app/files.js

```diff
@@ -88,11 +88,15 @@ module.exports.CUSTOM_TEXT_STYLES = `${module.exports.CUSTOM_TEXT_PATH}/styles.j
 
 // COMPONENTS - CUSTOM TEXT INPUT
 module.exports.CUSTOM_TEXT_INPUT_PATH = `${module.exports.COMPONENTS_PATH}/CustomTextInput`;
-module.exports.CUSTOM_TEXT_INPUT = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/index.js`;
-module.exports.CUSTOM_TEXT_INPUT_STYLES = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/styles.js`;
+module.exports.CUSTOM_TEXT_INPUT = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/index.tsx`;
+module.exports.CUSTOM_TEXT_INPUT_STYLES = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/styles.ts`;
 module.exports.CUSTOM_TEXT_INPUT_ASSETS_PATH = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/assets`;
 module.exports.CTI_SHOW_PASSWORD_PATH = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/components/ShowPassword`;
 
+// COMPONENTS - ANIMATED CUSTOM TEXT INPUT
+module.exports.ANIMATED_CUSTOM_TEXT_INPUT_PATH = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/AnimatedCustomTextInput`;
```

Comment:
why are you setting into the CustomTextInput? I think it's better to be a new generic component, or if you want to set it inside CustomTextInput then let's continue using the folder structure as we are used to

Reply:
We talked this with Santi, Both component Share the ShowPassword component and a few assets, it's because of that I took that decision